packages/block-editor/src/components/font-sizes/with-font-sizes.js

```diff
@@ -160,35 +155,40 @@ export default ( ...fontSizeNames ) => {
 						}
 
 						const newState = Object.entries(
-							pickBy(
-								fontSizeAttributeNames,
-								didAttributesChange
+							fontSizeAttributeNames
+						)
+							.filter( ( [ key, value ] ) =>
```

Comment:
This is safe to do, since we've already converted to an array, and there's a check for emptiness just a few lines above.